charts/seed-monitoring/charts/prometheus/templates/prometheus.yaml

```diff
@@ -186,6 +184,26 @@ spec:
         - mountPath: /var/run/secrets/blackbox
           name: prometheus-kubeconfig
           readOnly: true
+      - name: prometheus-config-reloader
+        image: {{ index .Values.images "configmap-reloader" }}
+        imagePullPolicy: IfNotPresent
+        args:
+        - -webhook-url=http://localhost:9090/-/reload
+        - -volume-dir=/etc/prometheus/config
+        - -volume-dir=/etc/prometheus/rules
+        resources:
+          requests:
+            cpu: 5m
+            memory: 10Mi
+          limits:
+            cpu: 10m
+            memory: 20Mi
+        volumeMounts:
+        - mountPath: /etc/prometheus/config
+          name: config
+          readOnly: true
+        - mountPath: /etc/prometheus/rules
```

Comment:
Why is the `rules` mount not readonly?

Reply:
I copied this from the prometheus in `seed-bootstrap`. I'll change `rules` to readOnly in both Prometheus charts.